src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -54,7 +54,7 @@ const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
         <FastImage source={picture} style={[styles.picture, styles.anonBgColor]} />
       )}
       <Text numberOfLines={1} ellipsizeMode="tail" style={styles.name}>
-        as {name}
+        {type === ANONYMOUS ? name : `as ${name}`}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text rendering logic has been updated to conditionally display the name based on the `type`. If the type is `ANONYMOUS`, it will just display the `name`, otherwise, it will display `as name`. This change is necessary for handling anonymous chats.
```diff
- as {name}
+ {type === ANONYMOUS ? name : `as ${name}`}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->